content/docs/introduction/getting_started.md

```diff
@@ -7,7 +7,7 @@ sort_rank: 3
 
 This guide is a "Hello World"-style tutorial which shows how to install,
 configure, and use Prometheus in a simple example setup. You will download and run
-Prometheus locally, configure it to scrape itself and an example application,
+Prometheus locally. You'll configure it to scrape itself and an example application,
```

Comment:
We try to avoid contractions. I think the previous version reads a little better.

Reply:
Sure. I changed it because it's a bit of a long sentence but I'll go back to the previous iteration.